[IMP] sales: add pdfs to quotes page update (#17296)
odoo/documentation: content/applications/sales/sales/sales_quotations/pdf_quote_builder/add_pdf_quotes.rst

> Add PDFs to quotes

Reviewer: ```suggestion
Add PDFs to a quote
```

Reviewer: to match _Add PDFs to a product_

or update the other to match this one

==================

> Navigate to :menuselection:`Sales app --> Quotations` and select the desired quotation. Click the

Reviewer: consider adding a heading for _add pdfs to a quote_, since you have one for _quote template_ as well. just to clarify there are TWO sections, and if it generates the article toc too that would be a nice bonus

:guilabel:`Quote Builder` tab, and all available headers and footers are displayed in their own self

> titled sections. If any products have PDFs attached on their form, then a :guilabel:`Product`

Reviewer: self-titled

section displays.

Select the additional PDFs to be merged into the final PDF. If a selected PDF contains :ref:`custom
note form fields <pdf_quote_builder/dynamic_text/custom-note>`, they appear as editable text boxes
to be filled in.

.. image:: add_pdf_quotes/quote-builder-headers.png

>    :alt: A quote's Quote Builder tab with selected headers and footers. Custom note form fields are

Reviewer: ```suggestion
   :alt: The Quote Builder tab with selected headers and footers. Custom note form fields are
```

    displayed.

> Odoo provides the option to print a PDF of the quote for record keeping. To print the PDF quote,

Reviewer: ```suggestion
The final PDF quote can be downloaded and printed for record-keeping. To print the PDF quote,
```

navigate to the confirmed quote, and click the :icon:`fa-cog` :guilabel:`(Actions)` icon to reveal a
drop-down menu. From this drop-down menu, select :guilabel:`Print`, then select :guilabel:`PDF

> Quote`.

Reviewer: i think this could be a `menuselection` right? since it's all in one click

.. image:: add_pdf_quotes/drop-down-print-pdf.png

>    :alt: Print pdf quote option on drop-down menu located on confirmed sales order in Odoo Sales.

Reviewer: ```suggestion
   :alt: A sales order drop-down menu with the option to Print PDF Quote.
```

The PDF automatically downloads. When opened, the PDF quote, along with the configured header and

> footer can be viewed and printed.

Reviewer: ```suggestion
Open the PDF quote file to confirm the configured header,
footer, and additional PDFs before printing.
```

.. note::
   Download these :download:`PDF quote builder examples
   <add_pdf_quotes/pdfquotebuilderexamples.zip>` or download :download:`sample quotation

>    <add_pdf_quotes/sample_quotation.pdf>` for added reference.

Reviewer: ```suggestion
<add_pdf_quotes/pdfquotebuilderexamples.zip>` or this :download:`sample quotation
<add_pdf_quotes/sample_quotation.pdf>` for additional reference.
```

.. example::

>    A car dealership wants to add their vehicle purchase order form as a header to their quotes in a

Reviewer: ```suggestion
   A car dealership wants to include its vehicle purchase order form with each quotation.
```

>    move to digitize all their documents. They create a PDF template of the form and upload it to

Reviewer: ```suggestion
   They create a PDF template of the form and upload it to
```

>    **Sales** app as a header.

Reviewer: ```suggestion
   the **Sales** app as a header.
```

   When they create a quote for a customer, they select the header in the Quote Builder tab. From

>    the tab, the car seller fills in all the details manually in the customized note form fields.

Reviewer: ```suggestion
   When they create a quote for a customer, they select the header in the *Quote Builder* tab and manually fill in the customized note form fields.
```

   .. image:: add_pdf_quotes/example-voc-header.png

>       :alt: The vehicle purchase order form selected in the Quote Builder tab of a quote.

Reviewer: ```suggestion
      :alt: The Quote Builder tab with a vehicle purchase order form selected as the header.
```

>    When the quote is signed and confirmed, the car seller prints a copy of the PDF quote for the

Reviewer: ```suggestion
   After the quote has been signed and confirmed, the car seller downloads and prints a copy of the PDF quote for the
```

   customer. The PDF quote contains the vehicle purchase order form as the header, the quote details
   in the body, and the dealership's standard footer.

> Add PDFs to quote templates

Reviewer: ```suggestion
Add PDFs to a quote template
```

===========================

.. important::

>    Headers and footers added to a quote template are **only** available when that template is

Reviewer: ```suggestion
   Headers and footers added to a quote template are only available when that template is
```

Reviewer: totally optional! i prefer to avoid formatting, but feel free to leave it in if you want to emphasize

   applied — they won't appear as standalone header/footer options for quotes.

To add PDFs to a quote template, navigate to :menuselection:`Sales app --> Configuration --> Quote
Templates`, and select the desired template. Click the :guilabel:`Quote Builder` tab and click

> :guilabel:`Add Headers and footers`. In the *Add: Headers and footers* pop-up, select the desired

Reviewer: ```suggestion
:guilabel:`Add Headers and footers`. In the *Add: Headers and footers* pop-up, select the 
```

Reviewer: alt. remove "to be added", the two are redundant

> PDF to be added to the quote template. Any selected PDFs display within the :guilabel:`Quote

Reviewer: ```suggestion
PDF to be added to the quote template. Selected PDFs are displayed in the :guilabel:`Quote
```

Builder` tab as cards.

To remove a header or footer, click the :icon:`fa-ellipsis-v` :guilabel:`vertical ellipsis` icon of

> a card and select :guilabel:`Delete`. Deleting a header or footer from a quote template only removes

Reviewer: ```suggestion
To remove a header or footer, click the :icon:`fa-ellipsis-v` :guilabel:`vertical ellipsis` icon on
the card and select :guilabel:`Delete`. Deleting a header or footer from a quote template only removes
```

> it from the template. It's still available in the *Quote Builder* tab of quotes or on the

Reviewer: ```suggestion
it from the template. It remains available in the *Quote Builder* tab and the
```

Reviewer: alt. i think this is one of the situations where it's fine to use future tense, i.e. "it will still be available"
